src/views/Profile/PersonalInfo/RegionCityHandler.tsx

```diff
@@ -151,6 +155,11 @@ const RegionCityHandler = ({
           disablePortal
           options={selectCities || cities}
           fullWidth
+          value={
+            newAuthorValues.cityId
+              ? cities.find((city) => city.id === newAuthorValues.cityId)
```

Comment:
I think ternary is not really needed here:

```
value={cities.find((city) => city.id === newAuthorValues.cityId)}
```

will return `undefined` if `newAuthroValues.cityId` is `null`, `undefined` or empty string